web/profiles/custom/os2loop/themes/os2loop_theme/templates/content/node--os2loop-documents-document.html.twig

```diff
@@ -88,66 +92,23 @@
-    </div>
-  </div>
-</div>
-  <div class="icon-container">
-    <div class="icon print" title="{{ 'Print document (PDF)'|t }}">
-      <a aria-label="{{ 'Print document (PDF)'|t }}" href="{{ path('entity_print.view', {export_type: 'pdf', entity_type: 'node', entity_id: node.id}) }}"></a>
-    </div>
-    <div class="icon flag" title="{{ 'contact editorial office'|t }}">
-      <a aria-label="{{ 'Contact editorial staff'|t }}" href="{{ path('os2loop_flag_content.flag_content_form', {node: node.id}) }}"></a>
-    </div>
-    <div class="icon share" title="{{ 'Share with a friend'|t }}">
-      <a aria-label="{{ 'Share with a friend'|t }}"></a>
-    </div>
-     <div class="icon edit" title="{{ 'Edit document'|t }}">
-        <a aria-label="{{ 'Edit document'|t }}" href="{{ path('entity.node.edit_form', {node: node.id}) }}"></a>
-      </div>
-      <div class="icon delete" title="{{ 'Delete document'|t }}">
-        <a aria-label="{{ 'delete document'|t }}" href="{{ path('entity.node.delete_form', {node: node.id}) }}"></a>
-      </div>
-      {{ content.flag_os2loop_favourite }}
-  </div>
-</div>
-{% endmacro %}
-
 <article{{ attributes }}>
-  <ul class="tabs nav" id="document-tabs" role="tablist">
-    <li>
-      <a class="tab active" id="document-tab" data-toggle="tab" href="#document" role="tab" aria-controls="document" aria-selected="true">{{ "Document"|t }}</a>
-    </li>
-    <li>
-      <a class="tab" id="about-document-tab" data-toggle="tab" href="#about-document" role="tab" aria-controls="about-document" aria-selected="false">{{ "About document"|t }}</a>
-    </li>
-    <li>
-      <a class="tab" id="about-collection-tab" data-toggle="tab" href="#about-collection" role="tab" aria-controls="about-collection" aria-selected="false">{{ "Document collection"|t }}</a>
-    </li>
-  </ul>
+  {% set tabs = [
+    {'id': 'document', 'name': 'Document'|t},
+    {'id': 'about-document', 'name': 'About document'|t},
+    {'id': 'about-collection', 'name': 'Document collection'|t}
+  ] %}
+  {% include '@os2loop_theme/navigation/content-tabs.html.twig' with {'tabs': tabs, 'id': 'document-tabs'} %}
   <div class="tab-content" id="document-tabs-content">
     <div class="tab-pane show active" id="document" role="tabpanel" aria-labelledby="document-tab">
-      {{ title_prefix }}
-      {% if label  %}
-        {{ _self.title(label, "document", node, content) }}
-      {% endif %}
+      {{ macros.title(label, "document", node, content) }}
       <div{{ content_attributes }}>
         {{ content.os2loop_documents_document_body }}
         {{ content.os2loop_documents_document_conte }}
       </div>
     </div>
     <div class="tab-pane" id="about-document" role="tabpanel" aria-labelledby="about-document-tab">
-      {% if label  %}
-        {% set title = "About document"|t %}
-        {{ _self.title(title, "document", node, content) }}
-      {% endif %}
+      {{ macros.title("About document", "document", node, content) }}
```

Comment:
`"About document"` must be translated, i.e. `{{ macros.title("About document"|t, … }}`.

Reply:
Fixed